aries-site/src/examples/components/pagination/PaginationServerTableExample.js

```diff
@@ -104,35 +116,50 @@ export const PaginationServerTableExample = () => {
       </Heading>
 
       <Box>
-        <DataTable
-          aria-describedby="server-side-pagination-heading"
-          columns={columns}
-          data={data}
-          sort={{ ...sort, external: true }}
-          step={limit}
-          onSort={opts => setSort(opts)}
-          fill
-          gap="small"
-        />
+        <Box overflow="auto">
+          <DataTable
+            aria-describedby="server-side-pagination-heading"
+            columns={columns}
+            data={data}
+            sort={{ ...sort, external: true }}
+            onSort={opts => setSort(opts)}
+            fill
+            gap="small"
+          />
+        </Box>
 
         {numberItems > limit && (
           <Box
             direction="row-responsive"
-            fill="horizontal"
             border="top"
-            justify="end"
-            pad={{ vertical: 'xsmall' }}
+            gap="medium"
+            pad={{ vertical: 'small' }}
+            align="center"
+            wrap
           >
-            <Text>
-              Showing {pageResultStart}-{pageResultEnd} of {numberItems}
-            </Text>
+            {!mobile && (
+              <Box direction="row-responsive" gap="medium" align="center">
+                <Text>
+                  Showing {pageResultStart}-{pageResultEnd} of {numberItems}
```

Comment:
I was expecting that the "Show X of Y" text would be on its own and the "Rows per page" would be grouped in a Box with the Pagination control